lib/matplotlib/cbook/deprecation.py

```diff
@@ -221,9 +217,11 @@ def wrapper(*args, **kwargs):
 
         old_doc = textwrap.dedent(old_doc or '').strip('\n')
         message = message.strip()
-        new_doc = (('\n.. deprecated:: %(since)s'
-                    '\n    %(message)s\n\n' %
-                    {'since': since, 'message': message}) + old_doc)
+        new_doc = ('.. deprecated:: {since}\n'
```

Comment:
What happened to the leading newline?

Reply:
It's at the beginning of the docstring so it doesn't matter.

OTOH I realized that we should dedent old_doc with inspect.cleandoc instead of textwrap.dedent, as the first line may not be indented... e.g. right now (before this PR) the docstring of `backend_cairo.ArrayWrapper.__init__` is
```
.. deprecated:: 3.0
    The ArrayWrapper class was deprecated in Matplotlib 3.0 and will be removed in 3.2.

Thin wrapper around numpy ndarray to expose the interface
       expected by cairocffi. Basically replicates the
       array.array interface.
```